website/src/pages/playground/index.js

```diff
@@ -18,7 +18,7 @@ import {
 import useDebounce from "../../hooks/useDebounce";
 import {
   getReactLiveNoInline,
-  annotateCodeForPlayground,
+  // annotateCodeForPlayground,
```

Comment:
Since this code is not heavily used, commenting out for this PR. 
Will get back to investigating this feature in another pr.